src/int.rs

```diff
@@ -108,11 +104,6 @@ impl Int {
         Int::from(ApInt::zero(width))
     }
 
-    /// Creates a new `Int` with the given bit width that represents one.
-    pub fn one(width: BitWidth) -> Int {
-        Int::from(ApInt::one(width))
-    }
```

Comment:
I think it is okay to drop `one` and `zero` constructors for `Apint`, however for `Int` and `Uint` I think they make sense since those types actually represent signed or unsigned integers and one or zero are very typical values that are often in use.

Reply:
The `zero` constructor is still in `apint`